sound/soc/sof/mediatek/adsp_pcm.c

```diff
@@ -0,0 +1,237 @@
+// SPDX-License-Identifier: (GPL-2.0-only OR BSD-3-Clause)
+//
+// Copyright(c) 2021 Mediatek Corporation. All rights reserved.
+//
+// Author: Bo Pan <bo.pan@mediatek.com>
+//		YC Hung <yc.hung@mediatek.com>
+//
+// The Mediatek ADSP PCM implementation
+//
+
+#include "adsp_pcm.h"
+#include "adsp_helper.h"
```

Comment:
"you're including this twice, remove here."

this is the inclusion, so you don't need to have #include "adsp_pcm.h" in "adsp_helper.h"